lively-lions/MUD/items/views/item_view.py

```diff
@@ -0,0 +1,36 @@
+from django.http import JsonResponse
```

Comment:
is the mix of class-based views and function-based views due to different team members writing code?